tests/controllers/home.test.js

```diff
@@ -28,17 +29,21 @@ test("home GET without breach renders monitor without breach", () => {
 });
 
 
-test("home GET with breach renders monitor with breach", () => {
+test("home GET with breach renders monitor with breach", async() => {
   const testBreach = {Name: "Test"};
   mockRequest.query = { breach: testBreach.Name };
   mockRequest = addBreachesToMockRequest(mockRequest);
-  mockRequest.url = "https://www.mozilla.com";
   mockRequest.session = { user: null };
-  const mockResponse = { render: jest.fn() };
+  mockRequest.url = { url: "https://www.mozilla.com" };
+  mockRequest.app.locals.SERVER_URL = AppConstants.SERVER_URL;
 
 
+  const mockResponse = { render: jest.fn(), redirect: jest.fn() };
   home.home(mockRequest, mockResponse);
+  const scanRes = await scanResult(mockRequest);
```

Comment:
We could move this test code into a new test module that only tests the `scanResult` function in isolation from the rest of this test logic.